config/boards/rock-5b-plus.csc

```diff
@@ -3,7 +3,7 @@ BOARD_NAME="Rock 5B Plus"
 BOARDFAMILY="rockchip-rk3588"
 BOARD_MAINTAINER=""
 BOOTCONFIG="rock-5b-plus-rk3588_defconfig"
-KERNEL_TARGET="vendor"
+KERNEL_TARGET="edge,vendor"
 FULL_DESKTOP="yes"
 BOOT_LOGO="desktop"
 BOOT_FDT_FILE="rockchip/rk3588-radxa-rock-5b+.dtb"
```

Comment:
Upstream name will likely be `rk3588-rock-5b-plus`, same as in radxa tree. Suggest a rename to match the most likely upstream name.

Reply:
I vaguely remember things breaking with vendor u-boot for some reason when using a different name, but I might be remembering wrong.

@HeyMeco do you know anything more?

Reply:
@fridtjof 
Changing the name from BOOT_FDT_FILE would be no issue (that's for the kernel). As long as BOOTCONFIG (that tells U-Boot what to use) stays as is and the name change is also being done in armbian/linux-rockchip

Reply:
Would overriding it in the post_family_config_branch_edge hook for now work as well? Then this PR does not implicitly break the vendor kernel until it's changed there (could be done in a trivial PR to change it on both sides then)

Reply:
I tested both builds like that and it seemed to work like i intended, so that should be fine now. I'll send a PR to the rockchip tree to fix the DT there later then. Resolve if that's okay :)